thunx-pdp-opa/src/main/java/com/contentgrid/thunx/pdp/opa/QuerySetToThunkExpressionConverter.java

```diff
@@ -93,7 +95,8 @@ static class PredicatesVisitor implements RegoVisitor<ThunkExpression<?>> {
                 Map.entry("gt", Comparison::greater),
                 Map.entry("gte", Comparison::greaterOrEquals),
                 Map.entry("lt", Comparison::less),
-                Map.entry("lte", Comparison::lessOrEquals)
+                Map.entry("lte", Comparison::lessOrEquals),
+                Map.entry("internal.member_2", Comparison::in)
```

Comment:
Because this is a weird name, can you reference the OPA source in a comment?
https://github.com/open-policy-agent/opa/blob/4c31c81a0fea2cb648f6c1f6b6ecd7a5e867e3f7/v1/ast/builtins.go#L368-L380

Reply:
Still missing a comment here :smile: